.github/workflows/windows.yml

```diff
@@ -64,6 +64,12 @@ jobs:
           git -c advice.detachedHead=false checkout $baseline
           bootstrap-vcpkg.bat -disableMetrics
 
+      - name: Set up vcpkg cache
+        uses: ./.github/actions/cache-vcpkg
```

Comment:
minor: Why the `./` prefix?

Reply:
That's how it's written in the help page you referenced.